frontend/vue/components/Syllabus/SyllabusView.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <section class="syllabus-view">
+    <h1 class="syllabus-view__title">
+      {{ syllabus.title }}
+    </h1>
+    <h2 class="syllabus-view__section-title">
+      General Information
+    </h2>
+    <SyllabusGeneralInformation class="syllabus-view__general-information">
+      <template #instructor>
+        {{ syllabus.instructor }}
+      </template>
+      <template #location>
+        {{ syllabus.location }}
+      </template>
+      <template #university>
+        {{ syllabus.university }}
+      </template>
+      <template #office-hours>
+        {{ syllabus.officeHours }}
+      </template>
+      <template #class-hours>
+        {{ syllabus.classHours }}
+      </template>
+      <template #email>
+        {{ syllabus.email }}
+      </template>
+    </SyllabusGeneralInformation>
+    <div
+      v-for="(workPackage, idx) in syllabus.workPackages"
+      :key="`${idx}-${workPackage.title}`"
+      class="syllabus-view__work-package"
+    >
+      <div class="syllabus-view__work-package__title">
+        {{ workPackage.title }}
+      </div>
+      <p class="syllabus-view__work-package__description">
+        {{ workPackage.description }}
+      </p>
+      <div class="syllabus-view__work-package__chapters-title">
+        <!-- TODO replace with translation on final design -->
+        Chapters
+      </div>
+      <ColumnFlowGrid
+        tag="ul"
+        class="syllabus-view__work-package__chapter-list"
+        :elements="workPackage.chapterList"
+      >
+        <template #default="slotProps">
+          <li class="syllabus-view__work-package__chapter-list__item">
+            <BasicLink class="syllabus-view__work-package__chapter-list__chapter" :url="slotProps.element.url">
```

Comment:
One small thing I noticed is in the URL structure for the course links... 

I think `../syllabus/course/...` needs to be updated to remove `syllabus`

![Screen Shot 2022-03-10 at 11 34 09 AM (2)](https://user-images.githubusercontent.com/6276074/157741184-d4a1df75-b709-421c-b561-45e06e65195d.png)

Reply:
Yeah, thanks!
It is an issue related with the mock data. It should be fixed now